src/internal/events/events.go

```diff
@@ -191,7 +191,9 @@ func tenantHash(tenID string) string {
 type m string
```

Comment:
nit: Can we change `m` to something like `metricCategory` for better readability?